src/pages-and-resources/pages/PageGrid.jsx

```diff
@@ -2,12 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import PageCard from './PageCard';
-
 import messages from '../messages';
 
 function PageGrid({ intl, pages }) {
   return (
-    <div>
+    <div id="course-pages-grid">
```

Comment:
I'd only bother to add IDs to things if we need to use the ID somewhere; not seeing any usages of it in the PR.

Reply:
@davidjoy  Yeah I initially added this because I wanted to reference it in Modal as the parent component. I'm not doing that any more so I'll remove this.